test/Interop/Cxx/class/constructors-executable.swift

```diff
@@ -1,9 +1,10 @@
-// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -Xfrontend -disable-availability-checking)
+// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -cxx-interoperability-mode=default -Xfrontend -enable-experimental-feature -Xfrontend CXXForeignReferenceTypeInitializers -Xfrontend -disable-availability-checking)
```

Comment:
Hmm, I think we shouldn't need `-Xfrontend` before `-enable-experimental-feature`, since `-enable-experimental-feature` should also be a driver-level flag.